### PR TITLE
feat: CHAPI wizard custom empty state handling

### LINKED ISSUE #293: CHAPI wizard : customize empty state page when wallet not registered

### PR DESCRIPTION
**Pull request by sudeshrshetty**
- closes #293

Signed-off-by: sudesh.shetty <sudesh.shetty@securekey.com>

### CODE REVIEW

**Review by fqutishat — APPROVED** [review]

**Comment by sudeshrshetty** [comment]
image

image

File: cmd/credential-mediator/components/Mediator.vue [hunk 1]
```diff
@@ -197,6 +205,12 @@
 </template>
 
 <script>
+    /*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
```
Comment (llorllale) [inline cmd/credential-mediator/components/Mediator.vue:209]: should this not specify the year range? (see the date range DB added below)

Comment (llorllale) [reply cmd/credential-mediator/components/Mediator.vue:209]: Something like "Portions of this software Copyright 2020, The TrustBloc contributors"

Comment (llorllale) [reply cmd/credential-mediator/components/Mediator.vue:209]: @troyronda ^^

**Pull request merged**